src/test/kotlin/ecommerce/CRUDTest.kt

```diff
@@ -0,0 +1,160 @@
+package ecommerce
+
+import io.restassured.RestAssured
+import io.restassured.http.ContentType
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.http.HttpStatus
+import org.springframework.jdbc.core.JdbcTemplate
+import org.springframework.test.annotation.DirtiesContext
+
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+class CRUDTest {
+    private lateinit var productRepository: ProductRepository
+
+    @Autowired
+    private lateinit var jdbcTemplate: JdbcTemplate
+
+    @BeforeEach
+    fun setUp() {
+        productRepository = ProductRepository(jdbcTemplate)
+
+        jdbcTemplate.execute("DROP TABLE products IF EXISTS")
+        jdbcTemplate.execute(
+            "CREATE TABLE products(" +
+                "id BIGINT AUTO_INCREMENT, name VARCHAR(255) NOT NULL, price DOUBLE NOT NULL, image_url VARCHAR(512) NOT NULL)",
+        )
```

Comment:
Why should we drop the table and create the same one **`BeforeEach` tests**?
I mean, we might as well reuse the one that we created before and save some time 🤔